types/block.yaml

```diff
@@ -1,52 +1,69 @@
 schemas:
+  BeaconBlockCommon:
```

Comment:
My bad on bumping to v0.11.0 links without fixing params.

Is the plan here to get v0.10.x cleaned up and release. Then update to v0.11?